### PR TITLE
[linux-port] Enable Travis Linux and macOS bots.

### PR DESCRIPTION
**Pull request by ehsannas**

### CODE REVIEW

**Comment by ehsannas** [comment]
hmm... looks like travis is not running...
Let's try without 'allow_failures'

**Comment by ehsannas** [comment]
hmm... looks like that was not the issue. Travis is probably not being triggered at all.

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+cache:
+  apt: true
+
+git:
+  depth: 1
+
+branches:
+  only:
+    - linux
```
[the remark this answers is not in the document]
Comment (ehsannas) [reply .travis.yml]: Oops

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,75 @@
+# Linux Build Configuration for Travis
```
Comment (antiagainst) [inline .travis.yml]: Linux and macOS

**Pull request merged**